packages/ui/WizardForm.tsx

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from "next/router";
+
+import Stepper from "./Stepper";
+
+type DefaultStep = {
+  title: string;
+  description: string;
+  content: JSX.Element;
+  enabled?: boolean;
+};
+
+function WizardForm<T extends DefaultStep>(props: { href: string; steps: T[] }) {
```

Comment:
Same v2 comment

Reply:
V2 Should be good to merge into main now - agreed these should live in V2

Reply:
We can't move, see comment above.